includes/bootstrap.inc

```diff
@@ -273,7 +274,7 @@ function drush_bootstrap_validate($phase) {
         break;
       }
       if ($phase_index > $validated_phase) {
-        $current_phase .= '_validate';
+        $current_phase .= 'Validate';
```

Comment:
If we are going to adopt all of PSR-2, then we must camelCase() our methods. This is definitely b/c breaking, but I think that now is a good time to just do it, as folks will hopefully be doing rewrites to annotated anyway. Drush 8 is still available for those who have trouble with the upgrade.